Gemfile

```diff
@@ -67,8 +63,9 @@ group(:omnibus_package) do
   # fixes.
   gem "chef", "= 15.8.23"
```

Comment:
Is there a reason not to bump chef to 15.10. If you unpin train and don't bump chef to 15.10 then bootstrapping doesn't work. This needs to get bumped to 15.10 and Ohai to 15.9 in order to provide a working knife experience.

Reply:
Another commit incoming that bumps this. The expeditor job failed because its missing a bunch of the other dep bumps that are in this PR, so I'm bringing it in here